index.html

```diff
@@ -2070,6 +2089,10 @@ <h3>Lifecycle Details</h3>
 any other actions involving a <a>credential</a>.
           </li>
           <li>
+In some cases, a <a>holder</a> might transfer to another <a>holder</a> one or more of its
+<a>verifiable credentials</a> that have not been bound by the <a>issuer</a> to that <a>holder</a>.
```

Comment:
Binding again.  Also appears to presume that association with Holder is default for VCs.